contracts/contracts/lib/AssetHelper.sol

```diff
@@ -10,8 +11,11 @@ import {ISubnetActor} from "../interfaces/ISubnetActor.sol";
 
 /// @notice Helpers to deal with a supply source.
 library AssetHelper {
+    using ExcessivelySafeCall for address;
     using SafeERC20 for IERC20;
 
+    uint16 constant private MAX_MEMORY_SIZE = 128;
```

Comment:
Can you elaborate on the attack vector (in the case of `IERC20.transfer` call)?

Reply:
probably the IERC20 implementation is custom and the behaviour is unknown, limit to how much it loads in memory.